include/modem_info.h

```diff
@@ -95,6 +95,7 @@ struct device_param {
 	const char *board; /**< Board version. */
 	const char *app_version; /**< Application version. */
 	const char *app_name; /**< Application name. */
+	double bat_charge; /**< The current battery charge, in percentage. */
```

Comment:
Do we need a floating point precision here? Isn't an integer in range 0 - 100 accurate enough (we deal with percents here, right?).